sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 5.2.0b4 (Unreleased)
 
-This version of the client library defaults to the latest supported API version, which currently is `2022-04-01-preview`.
+Note that this is the first version of the client library that targets the Azure Cognitive Service for Language APIs which includes the existing text analysis and natural language processing features found in the Text Analytics client library.
+In addition, the service API has changed from semantic to date-based versioning. This version of the client library defaults to the latest supported API version, which currently is `2022-04-01-preview`.
```

Comment:
Do you think it is worth mentioning that we remove support for 3.2-preview.2 but that functionalities are included in the new version?

Reply:
yeah let me add it